### PR TITLE
Systemd timesyncd

### PR DESCRIPTION
**Pull request by fgiudici**

### CODE REVIEW

**Review by mudler — APPROVED** [review]

File: packages/cloud-config/oem/09_services.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "Default config"
+stages:
+   initramfs:
+     - name: "Default systemd config"
+       if: '[ -e "/sbin/systemctl" ] || [ -e "/usr/bin/systemctl" ] || [ -e "/usr/sbin/systemctl" ] || [ -e "/usr/bin/systemctl" ]'
```
Comment (Itxaka) [inline packages/cloud-config/oem/09_services.yaml:5]: what is wrong with systemctl lol, can it be in 4 different places?? :laughing:

**Review by Itxaka — APPROVED** [review]

File: packages/cloud-config/oem/09_services.yaml (hunk 1 above)
Comment (mudler) [reply packages/cloud-config/oem/09_services.yaml:5]: argh yea, tried to be as much generic as possible here... maybe we should have some helpers in the system for such things :thinking:

Comment (kkaempf) [reply packages/cloud-config/oem/09_services.yaml:5]: The second one should be `/bin/systemctl`, no ?!
Otherwise the second and fourth are identical.

Comment (fgiudici) [reply packages/cloud-config/oem/09_services.yaml:5]: right... nice catch!

**Pull request merged**